docs/exception-mandate-for-tenant-level-api-path.md

```diff
@@ -0,0 +1,44 @@
+# ExceptionMandateForTenantLevelApiPath
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Uri-V1-11
+
+## Description
+
+Tenant level API's are not allowed for a PUT operation. Exception from PAS team is mandatory for the very first implementation. 
+Once the exception is granted author would need to suppress the error being flagged using https://github.com/Azure/autorest/blob/main/docs/generate/suppress-warnings.md#suppress-warnings
+
+## How to fix the violation
+
+The error can be fixed by one of the following ways:
+1. Do not define tenant level api's for PUT operation
+1. If tenant level PUT operation is mandatory, get exception from PAS team and then apply supression to the error
```

Comment:
>If tenant level PUT operation is mandatory, get exception from PAS team and then apply supression to the error [](http://example.com/codeflow?start=3&length=110)

If tenant level write operations cannot be avoided, get an exception from PAS team. Please reach out to the following contacts to do a design review - SARBACRev, prasnal@microsoft.com, ARM Auth Team <armrbac@microsoft.com>. Apply a suppression for this error providing an evidence of a sign-off to the API reviewer over email.